platform/embed/partner-solutions.mdx

```diff
@@ -1,11 +1,12 @@
 ---
 title: Partner Solutions Documentation
-description: "Heads up, our [Embed docs](https://docs.api.zapier.com/guides/intro) have moved."
+description: "Heads up, our [Embed docs](/partner-solutions/getting-started) have moved."
```

Comment:
I think if we're making changes here we should remove the old "moved" language entirely. Might be simple enough to just replace this description with a 1-liner for partner-solutions. Could potentially take the first sentence from below.